flink-ml-lib/src/main/java/org/apache/flink/ml/classification/logisticregression/LogisticRegression.java

```diff
@@ -362,7 +362,6 @@ private void updateModel() {
         @Override
         public void onEpochWatermarkIncremented(
                 int epochWatermark, Context context, Collector<double[]> collector) {
-            // TODO: let this method throws exception.
```

Comment:
Could you also update this method and do not throw runtime exception in this method?

Reply:
hmm... why do we need to do that?

Reply:
Thanks for pointing out this issue. The issue has been fixed.